scripts/plotting/test_training_distribution.py

```diff
@@ -0,0 +1,46 @@
+import matplotlib.pyplot as plt
+import xarray as xr
+
+def plot_label_distribution_split(data, split_indices, column='Label', bins=20, colors=['blue', 'green'], alpha=0.7, titles=['Training Set Label Distribution', 'Test Set Label Distribution'], x_label='Label', y_label='Frequency', figsize=(12, 6)):
```

Comment:
maybe give `data` a more meaningful name and give it a type. also `split_indices` should have a type